content/discussions/quickstart.md

```diff
@@ -43,7 +43,7 @@ For more information about providing guidelines for your project, see "[Adding a
 
 ## Creating a new discussion
 
-Anyone with access to a repository can create a discussion.
+Any logged-in user who can view the repository can create a discussion.
```

Comment:
We usually refer to a "logged-in user" as an "authenticated user". This covers people who authenticate by logging in directly, as well as people who authenticate using other methods, like SSO.

```suggestion
Any authenticated user who can view a repository can create a discussion.
```

Reply:
Thank you!